glui_control.cpp

```diff
@@ -313,12 +319,18 @@ int GLUI_Control::char_width(char c)
 { /* Hash table for faster character width lookups - JVK
        Speeds up the textbox a little bit.
   */
-  int hash_index = c % CHAR_WIDTH_HASH_SIZE;
+  /*Killing this with fire.
```

Comment:
"The replacement" reduces the size by a factor of 10 and I only meant it as a stopgap for full removal. IT's very bad for kilobytes to be spread out everywhere. It's not an improvement cache-wise.

You should know this. That constant is 128 and so is exactly identical to what I did. it's not a hash, it's a pretend hash--perhaps an aspiration to something else--and at best is only preventing non-ASCII characters from overflowing (by replacing them with random characters that are not necessarily printable.)

I think you're f**king with me half the time.